docs_sphinx/introduction/install.rst

```diff
@@ -50,8 +48,8 @@ Installing Brian 2
     :ref:`installation_from_source` instead.
 
 You can either install Brian 2 in the Anaconda root environment, or create a
-new environment for Brian 2 (http://conda.pydata.org/docs/using/envs.html). The
-latter has the advantage that you can update (or not update) the dependencies
+new environment for Brian 2 (https://docs.anaconda.com/anaconda/navigator/tutorials/manage-environments/).
```

Comment:
Maybe rather use ``https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html``? We never talk about the Anaconda Navigator elsewhere.